Server/VikingVault.Services/AdminService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Text;
+using VikingVault.DataAccess;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class AdminService : IAdminService
+    {
+        private readonly VikingVaultDbContext _dbContext;
+
+        public AdminService(VikingVaultDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public bool IsAdmin(string token)
+        {
+            var tokenObject = new JwtSecurityToken(token);
+            string userId = tokenObject.Payload["Id"].ToString();
+            var returnedUser = _dbContext.User.SingleOrDefault(u => u.Id == Int32.Parse(userId));
```

Comment:
When the return type is clear, it's okay to use `var`, for example in `var tokenObject = new JwtSecurityToken(token);`, but when it's not, like this `var returnedUser = _dbContext.User.SingleOrDefault(u => u.Id == Int32.Parse(userId));`, it's better to use an explicit type instead of `var`